scripts/browserstack.py

```diff
@@ -0,0 +1,58 @@
+"""
+Submits the apks to BrowserStack to run the instrumentation tests.
+"""
+
+import os
+import shlex
+import subprocess
+from subprocess import PIPE
+import sys
+import json
+
+def appendData(command, dataUrl):
+    var = "file=@{}".format(dataUrl)
+    return command + " " + json.dumps(var)
+
+def buildTestCommand(appToken, testToken):
+    test = {}
+    test["devices"] = ["LG G5-6.0", "Google Nexus 5-4.4"]
+    test["app"] = appToken
+    test["deviceLogs"] = True
+    test["testSuite"] = testToken
+    return json.dumps(json.dumps(test))
+
+if __name__ == "__main__":
+
+    if "BROWSERSTACK_USERNAME" in os.environ:
+        userName = os.environ["BROWSERSTACK_USERNAME"]
+
+    if "BROWSERSTACK_PASSWORD" in os.environ:
+        password = os.environ["BROWSERSTACK_PASSWORD"]
+
+    if "BASE_LOCATION" in os.environ:
+        baseLoc = os.environ["BASE_LOCATION"]
+
+    debugApk = baseLoc + "commcare/debug/app-commcare-debug.apk"
+    testApk = baseLoc + "androidTest/commcare/debug/app-commcare-debug-androidTest.apk"
```

Comment:
Q: Does brwserstack support uploading bundles ?

Reply:
yeah, I noticed that sometime ago, seems like that's a new feature there. I'll change this to use app bundles instead.

Reply:
Updated it to use app bundles. Also, the tests will be failing since I removed that null check! So until that one is fixed, they'll all fail!

Reply:
But Imma gonna address that in a separate PR cuz that issue is coming to real users as well. So I'm good to merge this one unless you have any concerns.

Reply:
@ShivamPokhriyal  I don't think there is value in us merging tests which are going to fail for time being. So unless you think otherwise, I will hold them off until the issue is fixed, verify that this PR works successfully on top of that other PR that fixes this isse and merge it afterwards.

Reply:
Cool makes sense.

Reply:
@shubham1g5 Hey I was wrong about it earlier. The tests are passing. The `IntentTestRule` was the reason behind the npe crashes and when we removed it here https://github.com/dimagi/commcare-android/pull/2231#discussion_r450157788, crashes also disappear.

Reply:
wow, that's great.